prow/plugins/golint/golint.go

```diff
@@ -139,30 +139,49 @@ func newProblems(cs []github.ReviewComment, ps map[string]map[int]lint.Problem)
 
 // problemsInFiles runs golint on the files. It returns a map from the file to
 // a map from the line in the patch to the problem.
-func problemsInFiles(r *git.Repo, files map[string]string) (map[string]map[int]lint.Problem, error) {
+func problemsInFiles(r *git.Repo, files map[string]string) (map[string]map[int]lint.Problem, github.DraftReviewComment, error) {
 	problems := make(map[string]map[int]lint.Problem)
+	var lintErrorComment github.DraftReviewComment
```

Comment:
This should probably be a slice of comments so that we can comment on all files that cannot be linted.

Reply:
@cjwagner do we want to return as soon as we hit a lint error or keep going? If we are going to return right away, we don't really need a slice of comments.